packages/devtools_app/lib/src/screens/app_size/app_size_table.dart

```diff
@@ -13,19 +13,28 @@ import '../../ui/colors.dart';
 
 import 'app_size_controller.dart';
 
-const deferredLoadingSupport = false;
+//Temporary feature flag for deferred loading.
+const deferredLoadingSupport = true;
```

Comment:
Let's move this to the `app_size_controller` file. That way, the `isDeferredApp` bool could return true if it is a `deferredApp` and the support is enabled. This way we don't have to check both everywhere.